.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .idea
-lib/**/*
\ No newline at end of file
+lib/**/*
```

Comment:
There should be a newline at the end of the file to adhere to standard conventions.